src/main/java/gregtech/api/recipes/RecipeBuilder.java

```diff
@@ -819,7 +820,11 @@ public R append(Recipe recipe, int multiplier, boolean multiplyDuration) {
 
         this.EUt(multiplyDuration ? recipe.getEUt() : this.EUt + recipe.getEUt() * multiplier);
         this.duration(multiplyDuration ? this.duration + recipe.getDuration() * multiplier : recipe.getDuration());
-        this.parallel += multiplier;
+        if (this.parallel == 0) {
+            this.parallel = multiplier;
+        } else {
+            this.parallel *= multiplier;
+        }
```

Comment:
Is this correct? I would think that the number of parallels should always be the number of the multiplier. Would there ever be a case where a recipe that is already being paralleled is passed to this method?